templates/main/group_builds.html.ep

```diff
@@ -1,4 +1,5 @@
-% for my $key (reverse sort keys %$result) {
+% use Sort::Versions;
+% for my $key (reverse sort { versioncmp($a, $b) } keys %$result) {
```

Comment:
Mh.. not sure whether sorting should be done in controller or view. Shouldn't make a big difference since is is only 2 lines of code.